requirements.txt

```diff
@@ -0,0 +1,13 @@
+#### begin pypi dependencies ####
+pytest==6.2.2

```

Comment:
each dependency moved to requirements.txt file (python best practice) with fixed version to avoid api conflict (ex: deprecated)